web/template/partial/course/manage/edit-video-sections.gohtml

```diff
@@ -36,21 +35,24 @@
                 <span class="font-light">Unsaved Changes</span>
             </div>
         </header>
-        <form @submit.prevent="videoSectionController.publishNewSections()">
+        <form id = "new-section-form" @submit.prevent="videoSectionController.publishNewSections()">
             <div class="flex align-middle">
                 <input id="startHours"
+                       onkeyup="this.value=this.value.replace(/[^\d]/,'')"
```

Comment:
Maybe we could also filter out values not in the range. E.g., valid hours are 0 <= 23h. I noticed that I can still put in any number. This would need to be done for the specific ranges of `startHours`,  `startMinutes` and `startSeconds`. Could be something for a follow-up as well, but I guess it also kinda fits the PR scope.

Reply:
If we really want to overengineer this, we could check whether the timestamp is actually within the duration of the video. But this is future work.

Reply:
👍🏻  

Added 
```javascript
this.current.startMinutes < 32 && // I think 32h hours will be sufficient.
this.current.startMinutes < 60 &&
this.current.startSeconds < 60
```

Regarding the second comment: I think a follow-up PR would make sense.